test/sequential/test-inspector-contexts.js

```diff
@@ -22,7 +22,9 @@ async function testContextCreatedAndDestroyed() {
     const mainContextPromise =
         notificationPromise('Runtime.executionContextCreated');
 
-    session.post('Runtime.enable');
+    session.post('Runtime.enable', (err) => {
```

Comment:
Nit: can be simplified to `session.post('Runtime.enable', assert.ifError)`.

Reply:
Thanks, done!